src/server/api/routers/waitlist.ts

```
@@ -0,0 +1,223 @@
+
+      // Use max position + 1 to avoid race condition on concurrent inserts (M2)
+      const maxEntry = await ctx.db.waitlist.findFirst({
+        orderBy: { position: "desc" },
+        select: { position: true },
+      });
+      const position = (maxEntry?.position ?? 0) + 1;
+
+      await ctx.db.waitlist.create({
+        data: {
+          name: input.name,
+          email,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'email' should be before 'name'.

(sort-keys-fix/sort-keys-fix)